build/musicbrainz-dev/scripts/fetch-dump.sh

```diff
@@ -151,48 +181,70 @@ then
 	# Find latest database dump corresponding to search indexes
 
 	SEARCH_DUMP_DAY="${DUMP_TIMESTAMP/-*}"
-	"${WGET_CMD[@]}" --spider --no-remove-listing -P "$DB_DUMP_DIR" \
-		"${BASE_FTP_URL:-$BASE_DOWNLOAD_URL}/$DB_DUMP_REMOTE_DIR"
+	rm -f "$DB_DUMP_DIR/index.html"
+	"${WGET_CMD[@]}" --force-html -O "$DB_DUMP_DIR/index.html" -P "$DB_DUMP_DIR" \
+		"${BASE_DOWNLOAD_URL}/$DB_DUMP_REMOTE_DIR/"
+	cat "$DB_DUMP_DIR/index.html"
 	DUMP_TIMESTAMP=$(
-		grep -E "\\s${SEARCH_DUMP_DAY}-\\d*" "$DB_DUMP_DIR/.listing" \
-			| sed -e 's/\s*$//' -e 's/.*\s//'
+		sed -n "s#.*href=\"[^\"]*\\($SEARCH_DUMP_DAY-[0-9]*\\).*#\\1#p" \
+			"$DB_DUMP_DIR/index.html" | head -1
```

Comment:
Using index.html feels brittle and hacky - is there anything we could do make this better (e.g. create a machine readable directory listing).

Reply:
I agree - just wondered if this was something that had been discussed before.